.github/workflows/test.yml

```
@@ -0,0 +1,37 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  go:
+    runs-on: ubuntu-latest
+    steps:    
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.15'
+
+      - name: Lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.54
+          run: golangci-lint
+
+      - name: Display Go version
+        run: go version
```

```suggestion
      - name: Display Go version
        run: go version
```
is this step necessary to display go version? i mean we already know go verison.

---

Ok i will remove➖ it.